docs/windows.md

```diff
@@ -70,26 +70,45 @@ choco install visualstudio2022-workload-vctools
 
 #### Install the Latest Python
 
-Download and install the latest version of Python:
+In Windows, The full installer contains all the Python components and is the
+best option for developers using Python for any kind of project.
+
+For more information on how to use the official installer, please see the
+[full installer instructions](https://docs.python.org/3/using/windows.html#windows-full).
+The default installation options should be fine for use with Gaphor.
+
+1. Install the latest Python version using the
+[official installer](https://www.python.org/downloads/windows/).
+
+2. Open a PowerShell terminal as a normal user and check the python version:
+
+   ```PowerShell
+   py -3.11 --version
+   ```
+
+#### Install Graphviz
+
+Graphviz is used by Gaphor for automatic diagram formatting.
 
 1. Install from Chocolately with administrator PowerShell:
 
    ```PowerShell
-   choco install python
+   choco install graphviz
```

Comment:
Maybe we should build graphviz as part of gvsbuild? So it does not ship with it's own libraries?

For now it's fine, though.

Reply:
Yes, I think so as well. I'll start looking in to this :+1: